presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -189,10 +304,14 @@ public void testRoundTripAllFormats(RoundTripTestCase testCase)
                         "all_datatypes_csv",
                         ImmutableList.of("f_bigint", "f_int", "f_smallint", "f_tinyint", "f_double", "f_boolean", "f_varchar"),
                         ImmutableList.of(100000, 1000, 100, 10, 1000.001, true, "'test'")))
+                .add(new RoundTripTestCase(
+                        "all_datatypes_json",
+                        ImmutableList.of("f_bigint", "f_int", "f_smallint", "f_tinyint", "f_double", "f_boolean", "f_varchar"),
+                        ImmutableList.of(100000, 1000, 100, 10, 1000.001, true, "'test'")))
```

Comment:
I think there was a suggestion in a previous review about testing with more than one record.

Reply:
yes, that commit is in the raw encoder pr #4417. I'll make sure that get's brought in when it merges